students/DiannaTingg/lessons/lesson03/assignment/test_basic_operations.py

```diff
@@ -0,0 +1,190 @@
+"""
+Tests for basic_operations
+"""
+
+import peewee as pw
+import pytest
+import customer_model as cm
+import basic_operations as bo
+
+customer1 = {"customer_id": "1", "first_name": "Fred", "last_name": "Flintstone",
+             "home_address": "301 Cobblestone Way, Bedrock, 70777", "phone_number": "928-635-2600",
+             "email_address": "fred@hotmail.com", "active_status": True, "credit_limit": 100.00}
+
+customer2 = {"customer_id": "2", "first_name": "Barney", "last_name": "Rubble",
+             "home_address": "303 Cobblestone Way, Bedrock, 70777", "phone_number": "928-635-2700",
+             "email_address": "barney@aol.com", "active_status": False, "credit_limit": 50.00}
+
+customer3 = {"customer_id": "3", "first_name": "Wilma", "last_name": "Flintstone",
+             "home_address": "301 Cobblestone Way, Bedrock, 70777", "phone_number": "928-635-2600",
+             "email_address": "wilma@hotmail.com", "active_status": True, "credit_limit": 100.00}
+
+customer4 = {"customer_id": "4", "first_name": "Betty", "last_name": "Rubble",
+             "home_address": "303 Cobblestone Way, Bedrock, 70777", "phone_number": "928-635-2700",
+             "email_address": "barney@aol.com", "active_status": False, "credit_limit": 50.00}
+
+
+def clear_database():
+    cm.database.drop_tables([
+        cm.Customer
+    ])
+
+    cm.database.close()
+
+
+def create_empty_database():
+    clear_database()
+
+    cm.database.create_tables([
+        cm.Customer
+    ])
+
+    cm.database.close()
```

Comment:
Consider making these two functions fixtures. Less typing, it automatically runs them before/after each test.
https://docs.pytest.org/en/latest/fixture.html